libr/anal/rtti_itanium.c

```diff
@@ -5,9 +5,9 @@
 #include <r_flag.h>
 #include <r_cons.h>
 
-#define vmi_class_type_info_name "obj.vtablefor__cxxabiv1::__vmi_class_type_info"
-#define class_type_info_name "obj.vtablefor__cxxabiv1::__class_type_info"
-#define si_class_type_info_name "obj.vtablefor__cxxabiv1::__si_class_type_info"
+#define VMI_CLASS_TYPE_INFO_NAME "obj.vtable_for___cxxabiv1::__vmi_class_type_info"
```

Comment:
defines shoulld  have a common/known/predicitible prefix to avoid colisions, but its fine to merge for now. this is probablly a task for @thestr4ng3r